src/battle_message.c

```diff
@@ -2229,7 +2241,19 @@ void BufferStringBattle(enum StringID stringID, u32 battler)
                 }
                 else
                 {
-                    if (gBattleTypeFlags & BATTLE_TYPE_MULTI)
+                    if (TESTING && gBattleTypeFlags & BATTLE_TYPE_MULTI)
+                    {
+                        if (gBattleScripting.battler == 1)
+                            stringPtr = sText_Trainer1SentOutPkmn;
```

Comment:
```suggestion
                        }
                            stringPtr = sText_Trainer1SentOutPkmn;
                        }
```
All parts of an if-else chain should have brackets if one of the parts have them.